lib/components/overlays/quick_settings/quick_settings_overlay.dart

```diff
@@ -117,6 +124,49 @@ class _QuickSettingsOverlayState extends State<QuickSettingsOverlay>
   }
 }
 
+PageRouteBuilder customRouteTransition(Widget screen) {
+  return PageRouteBuilder(
+    transitionDuration: const Duration(milliseconds: 150),
```

Comment:
I think it would make sense to use Constants.animationDuration here (and in the line after as well)

Reply:
I think it would make sense to have a constants.dart file in /utils for stuff like this.

Reply:
It's part of https://github.com/dahliaOS/shared